kolibri/auth/serializers.py

```diff
@@ -11,20 +11,7 @@ class Meta:
         model = Role
         exclude = ("dataset",)
 
-
-class FacilityUserSerializer(serializers.ModelSerializer):
-    roles = RoleSerializer(many=True, read_only=True)
-
-    class Meta:
-        model = FacilityUser
-        extra_kwargs = {'password': {'write_only': True}}
-        fields = ('id', 'username', 'full_name', 'password', 'facility', 'roles')
-
-    def create(self, validated_data):
-        user = FacilityUser(**validated_data)
-        user.set_password(validated_data['password'])
-        user.save()
-        return user
+class BaseKolibriUserSerializer(serializers.ModelSerializer):
```

Comment:
The `create` and `update` methods were identical between `FacilityUserSerializer` and `DeviceOwnerSerializer`. The common parts have been extracted to this shared base class.